app/assets/javascripts/components/course_creator/course_creator.jsx

```diff
@@ -65,7 +65,12 @@ const CourseCreator = React.createClass({
 
   storeDidChange() {
     this.setState(getState());
-    this.state.tempCourseId = CourseUtils.generateTempId(this.state.course);
+    if (this.state.course.term !== '' && this.state.course.school !== '' && this.state.title !== '') {
```

Comment:
The term field is not shown for the global WMF version of the course creator. This switches on the default course type (`this.state.default_course_type === 'ClassroomProgramCourse'`).

A slug valid without a term, it should just be shown without the trailing "_()".